packages/button/style.ts

```diff
@@ -187,21 +190,31 @@ export const button = {
   danger: filledButton(red, redDarken1, redDarken2)
 };
 
-export const buttonInverse = {
-  secondary: css`
-    ${tintContent(purpleLighten3)};
-    &:hover {
-      ${tintContent(purpleLighten2)};
-    }
-    &:active {
-      ${tintContent(purpleLighten1)};
-    }
-  `,
-  standard: filledButton(
-    hexToRgbA(white, 0.3),
-    hexToRgbA(white, 0.4),
-    hexToRgbA(white, 0.45)
-  )
+export const getButtonInverseStyles = (appearance: ButtonAppearances) => {
+  switch (appearance) {
+    case "secondary":
+      return css`
+        ${tintContent(purpleLighten3)};
+        &:hover {
+          ${tintContent(purpleLighten2)};
+        }
+        &:active {
+          ${tintContent(purpleLighten1)};
+        }
+      `;
+      break;
+    case "standard":
+      return css`
+        ${filledButton(
+          hexToRgbA(white, 0.3),
+          hexToRgbA(white, 0.4),
+          hexToRgbA(white, 0.45)
+        )};
+      `;
+      break;
+    default:
+      return "";
```

Comment:
is this intentional or do we might want to add a `error("Hey, you used a wrong appearance here!")`?

Reply:
It's not actually "wrong", it's just there's only inverse (dark mode) styles for the "secondary" and "standard" button appearances. So if `ButtonAppearances.Primary` gets passed, there's no styles to add